examples/gnrc_networking/README.md

```diff
@@ -1,5 +1,7 @@
 # gnrc_networking example
 
+You can try out the code in two different ways: You can either communicate between the RIOT machine and its Linux host, or you can communicate between two RIOT instances. Note that the former only works with native, i.e. if you run RIOT on your Linux machine.
```

Comment:
Oh and the wording is a little weird… Communication with a non-native node and Linux is possible through Ethernet, just not in the way it is descriped here.